lib/core/src/Cardano/Wallet/Primitive/Types/TokenBundle.hs

```diff
@@ -293,6 +294,13 @@ add (TokenBundle (Coin c1) m1) (TokenBundle (Coin c2) m2) =
 subtract :: TokenBundle -> TokenBundle -> Maybe TokenBundle
 subtract a b = guard (b `leq` a) $> unsafeSubtract a b
 
+difference :: TokenBundle -> TokenBundle -> TokenBundle
+difference (TokenBundle (Coin c1) m1) (TokenBundle (Coin c2) m2) =
+    TokenBundle
+        (Coin $ c1 - c2)
```

Comment:
Should add the properties for TokenMap to TokenBundle too. This `-` seems dangerous.

Reply:
Fixed.